rpcs3/Emu/RSX/Program/GLSLSnippets/RSXProg/RSXFragmentTextureOps.glsl

```diff
@@ -164,6 +169,15 @@ vec4 _texcoord_xform_shadow(const in vec4 coord4, const in sampler_info params)
 
 #endif // _EMULATE_SHADOW
 
+vec4 _sext_unorm8x4(const in vec4 x)
+{
+	// TODO: Handle clamped sign-extension
+	const vec4 bits = floor(fma(x, vec4(255.), vec4(0.5f)));

```

Comment:
.f

Reply:
Same thing in this case. The . vs .f thing dates back to the introduction of fp64 iirc and that feature has basically died out in mainstream GPU coding.